### DISCUSSION TITLE
Windows Installer "This App can't run on your PC"

### DISCUSSION THREAD

**Discussion by launchsolution** [oobabooga/text-generation-webui#240] [Q&A] [answered]
I am running windows 10 and the install throws this error any ideas?"This App can't run on your PC"

Input> A
"Downloading micromamba from https://github.com/cmdr2/stable-diffusion-ui/releases/download/v1.1/micromamba.exe to C:\Users\user\Documents\Chat\oobabooga\installer_files\mamba\micromamba.exe"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) schannel: next InitializeSecurityContext failed: Unknown error (0x80092012) - The revocation function was unable to check revocation for the certificate.
Micromamba version:
Access is denied.
Access is denied.
"Packages to install: torchvision torchaudio pytorch-cuda=11.7 conda git"
Access is denied.
Access is denied.
"There was a problem while installingtorchvision torchaudio pytorch-cuda=11.7 conda git using micromamba. Cannot continue."

**Comment by oobabooga** [comment]
Do you have an arm64 processor?

**Reply by launchsolution** [reply]
Device name	DESKTOP
Processor	Intel(R) Core(TM) i7-4790 CPU @ 3.60GHz   3.60 GHz
Installed RAM	16.0 GB (15.8 GB usable)
Device ID	0F2E495E-BFC1-4798-AE2C-FB76AEBA62E1
Product ID	00330-80000-00000-AA566
System type	64-bit operating system, x64-based processor
Pen and touch	No pen or touch input is available for this display

**Comment by JammersUK** [comment, marked as answer]
I had this error earlier.  The download "https://github.com/cmdr2/stable-diffusion-ui/releases/download/v1.1/micromamba.exe" is 0 bytes.  I just found the file on Github and copied it to the install directory.

**Reply by launchsolution** [reply]
Thanks that was driving me mad :)

**Comment by JammersUK** [comment]
When you first try to install via the windows script, it fails.  But when it fails it creates the directory: \oobabooga\installer_files\mamba That is the location to copy the micromamba.exe file into.  Just as long as that file isn’t 0kB, it should be around 8kB.
